simulatingrisk/hawkdove/model.py

```diff
@@ -199,4 +217,28 @@ def max_agent_points(self):
     def percent_hawk(self):
         # what percent of agents chose hawk?
         hawks = [a for a in self.schedule.agents if a.choice == Play.HAWK]
-        return len(hawks) / self.num_agents
+        phawk = len(hawks) / self.num_agents
+        # add to recent values
+        self.recent_percent_hawk.append(phawk)
+        return phawk
+
+    @property
+    def rolling_percent_hawk(self):
+        # make sure we have enough values to check
+        if len(self.recent_percent_hawk) > self.min_window:
+            rolling_phawk = statistics.mean(self.recent_percent_hawk)
+            # add to recent values
+            self.recent_rolling_percent_hawk.append(rolling_phawk)
+            return rolling_phawk
```

Comment:
What do you do for the first few moments without enough values? I guess the bar chart simply starts when rolling data is available?

Reply:
correct - the bar chart for % hawk each round displays, but the rolling mean doesn't display until we have one

(right now altair is doing a weird jump in the domain when the rolling mean starts that I haven't figured out)